sympy/integrals/tests/test_integrals.py

```diff
@@ -1893,6 +1893,16 @@ def test_issue_21024():
     assert F_true == integrate(f, x)
 
 
+def test_issue_21721():
+    a = Symbol('a')
+    b = Symbol('b', real=True)
+    expr = -I*(-2*I*pi*Heaviside(im(a) - 1, 0) + 2*I*pi*Heaviside(im(a) + 1, 0))/(2*pi)
+    assert integrate(1/(pi*(1+(x-a)**2)),(x,-oo,oo)) == expr
```

Comment:
if we put `integrate(1/(pi*(1+(x-a)**2)),(x,-oo,oo)).expand()` on the left hand side, them `expr` would look more readable.